executable/pe/64-bit-execution-via-heavens-gate.yml

```diff
@@ -0,0 +1,24 @@
+rule:
+  meta:
+    name: 64-bit-execution-via-heavens-gate
```

Comment:
filename uses dashes, but the rule name should use spaces and be ready for human display